build.gradle

```diff
@@ -49,20 +54,21 @@ buildscript {
 
     dependencies {
         classpath "com.android.tools.build:gradle:$android_gradle_plugin_version"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```

Comment:
TBH I'm not sure why things were working without this on our classpath, maybe we were just always using the "system" version of kotlin?